eclipse/pom.xml

```diff
@@ -519,6 +521,64 @@
 			</properties>
 		</profile>
 
+		<profile>
+			<id>updatesite-maven-aggregator</id>
+			<activation>
+				<file>
+					<exists>updatesite-aggr</exists>
+				</file>
+			</activation>
+			<build>
+				<resources>
+					<resource>
+					  <directory>updatesite-aggr</directory>
+					  <filtering>true</filtering>
+					</resource>
+				  </resources>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho.extras</groupId>
+						<artifactId>tycho-eclipserun-plugin</artifactId>
+						<version>${tycho.version}</version>
+						<executions>
+						  <execution>
+							<goals>
+							  <goal>eclipse-run</goal>
+							</goals>
+							<phase>package</phase>
+						  </execution>
+						</executions>
+						<configuration>
+						  <!-- DO NOT FORMAT THE FOLLOWING LINE, IT MUST BE ON ONE LINE -->
+						  <appArgLine>-application org.eclipse.cbi.p2repo.cli.headless aggregate -data target/workspace --buildModel target/classes/${updatesite.aggregator.filename}</appArgLine>
+						  <dependencies>
+							<dependency>
+							  <artifactId>org.eclipse.cbi.p2repo.aggregator.engine.feature</artifactId>
+							  <type>eclipse-feature</type>
+							</dependency>
+							<dependency>
+							  <artifactId>org.eclipse.equinox.core.feature</artifactId>
+							  <type>eclipse-feature</type>
+							</dependency>
+							<dependency>
+							  <artifactId>org.eclipse.equinox.p2.core.feature</artifactId>
+							  <type>eclipse-feature</type>
+							</dependency>
+						  </dependencies>
+						  <repositories>
+							<repository>
+							  <id>cbi-aggregator</id>
+							  <layout>p2</layout>
+							  <url>https://download.eclipse.org/cbi/updates/aggregator/headless/4.13/I20200625-1232/</url>
```

Comment:
I would favor a non-secure URL, so we can intercept the call with our reliability enhancing proxy.

We should find another solution for this at some point in time...

Reply:
Right. I forgot.